packages/ui/src/domainModules/collectionMammals/components/RecordNavigationBar/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon, Grid } from 'semantic-ui-react'
+
+export const RecordNavigationBar = ({
+  recordNumber,
+  totalRecords,
+  onPreviousRecord,
+  onNextRecord,
+  onShowAllRecords,
+  onNewRecord,
+}) => {
+  return (
+    <Grid textAlign="center">
+      <Grid.Column width={2}>
+        <Button
+          icon
+          onClick={() => onPreviousRecord({ recordNumber })}
```

Comment:
we can skip recordNumber here, just do `onPreviousRecord()` because the MammalManager knows the currentRecordNumber and will handle this